zm/zm_convr.F90

```diff
@@ -364,79 +361,79 @@ subroutine zm_convr_run(     ncol    ,pcols   ,pver    , &
+   real(kind_phys) zg(ncol,pver)             ! wg grid slice of gathered values of z.
+   real(kind_phys) sg(ncol,pver)             ! wg grid slice of gathered values of s.
+   real(kind_phys) tpg(ncol,pver)            ! wg grid slice of gathered values of tp.
+   real(kind_phys) zfg(ncol,pver+1)          ! wg grid slice of gathered values of zf.
+   real(kind_phys) qstpg(ncol,pver)          ! wg grid slice of gathered values of qstp.
+   real(kind_phys) ug(ncol,pver)             ! wg grid slice of gathered values of u.
+   real(kind_phys) vg(ncol,pver)             ! wg grid slice of gathered values of v.
+   real(kind_phys) cmeg(ncol,pver)
+
+   real(kind_phys) rprdg(ncol,pver)           ! wg gathered rain production rate
+   real(kind_phys) capeg(ncol)               ! wg gathered convective available potential energy.
+   real(kind_phys) tlg(ncol)                 ! wg grid slice of gathered values of tl.
+   real(kind_phys) landfracg(ncol)            ! wg grid slice of landfrac
+
+   integer lclg(ncol)       ! wg gathered values of lcl.
+   integer lelg(ncol)
 !
 ! work fields arising from gathered calculations.
 !
-   real(kind_phys) dqdt(pcols,pver)           ! wg mixing ratio tendency at gathered points.
-   real(kind_phys) dsdt(pcols,pver)           ! wg dry static energy ("temp") tendency at gathered points.
-!      real(kind_phys) alpha(pcols,pver)      ! array of vertical differencing used (=1. for upstream).
-   real(kind_phys) sd(pcols,pver)             ! wg grid slice of dry static energy in downdraft.
-   real(kind_phys) qd(pcols,pver)             ! wg grid slice of mixing ratio in downdraft.
-   real(kind_phys) mc(pcols,pver)             ! wg net upward (scaled by mb) cloud mass flux.
-   real(kind_phys) qhat(pcols,pver)           ! wg grid slice of upper interface mixing ratio.
-   real(kind_phys) qu(pcols,pver)             ! wg grid slice of mixing ratio in updraft.
-   real(kind_phys) su(pcols,pver)             ! wg grid slice of dry static energy in updraft.
-   real(kind_phys) qs(pcols,pver)             ! wg grid slice of saturation mixing ratio.
-   real(kind_phys) shat(pcols,pver)           ! wg grid slice of upper interface dry static energy.
-   real(kind_phys) hmn(pcols,pver)            ! wg moist static energy.
-   real(kind_phys) hsat(pcols,pver)           ! wg saturated moist static energy.
-   real(kind_phys) qlg(pcols,pver)
-   real(kind_phys) dudt(pcols,pver)           ! wg u-wind tendency at gathered points.
-   real(kind_phys) dvdt(pcols,pver)           ! wg v-wind tendency at gathered points.
-!      real(kind_phys) ud(pcols,pver)
-!      real(kind_phys) vd(pcols,pver)
+   real(kind_phys) dqdt(ncol,pver)           ! wg mixing ratio tendency at gathered points.
+   real(kind_phys) dsdt(ncol,pver)           ! wg dry static energy ("temp") tendency at gathered points.
+!      real(kind_phys) alpha(ncol,pver)      ! array of vertical differencing used (=1. for upstream).
```

Comment:
Should we go ahead and remove the commented-out variables here and below?

Reply:
deleted